src/common/com/intellij/plugins/haxe/compilation/HaxeCompilerUtil.java

```diff
@@ -270,47 +273,32 @@ public static int runInterruptibleCompileProcess(List<String> command,
                 timeLog.stamp("Executing " + command.get(0));
             }
             process = builder.start();
-
+            LOG.debug("Compiler process has started.");
             InputStreamReader stdoutReader = new InputStreamReader(process.getInputStream());
             BufferedReader bufferedStdout = new BufferedReader(stdoutReader);
 
             InputStreamReader stderrReader = mixedOutput ? null : new InputStreamReader(process.getErrorStream());
             BufferedReader bufferedStderr = mixedOutput ? null : new BufferedReader(stderrReader);
 
             do {
```

Comment:
The trouble here is that the output could come during the thread sleep at line 305/286.  If we stopped the loop at the end, the we wouldn't collect the output.  This change moves the collection to a subroutine and calls it again after the loop is complete so that we don't miss any.